Microsoft.Bot.Builder.sln

```diff
@@ -106,7 +106,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Schemas", "Schemas", "{EE56F2B6-4995-4E8F-ACFF-310AF0A4DA0F}"
 	ProjectSection(SolutionItems) = preProject
 		schemas\baseComponent.schema = schemas\baseComponent.schema
-		schemas\component.schema = schemas\component.schema

```

Comment:
>schemas\component.schema = schemas\component.schema [](start = 2, length = 51)

did you mean to remove this?

Reply:
It was there, but removed from project to regenerate and I forgot to add it back.  Fixed.